docs/admin/admission-controllers.md

```diff
@@ -111,9 +111,11 @@ so that quota is not prematurely incremented only for the request to be rejected
 
 This plug-in will observe the incoming request and ensure that it does not violate any of the constraints
 enumerated in the ```LimitRange``` object in a ```Namespace```.  If you are using ```LimitRange``` objects in
-your Kubernetes deployment, you MUST use this plug-in to enforce those constraints.
+your Kubernetes deployment, you MUST use this plug-in to enforce those constraints. LimitRanger can also
```

Comment:
Actually, we have multiple uses of both just within this doc. I'm going to leave this for a followup PR.